panel.js

```diff
@@ -1209,7 +1187,7 @@ var Panel = GObject.registerClass({
     }
 
     _onPanelMouseScroll(actor, event) {
-        let scrollAction = Me.settings.get_string('scroll-panel-action');
+        let scrollAction = SETTINGS.get_string('scroll-panel-action');
         let direction = Utils.getMouseScrollDirection(event);
 
         if (!this._checkIfIgnoredScrollSource(event.get_source()) && !this._timeoutsHandler.getId(T6)) {
```

Comment:
```suggestion
        const targetActor = global.stage.get_event_actor(event);

        if (!this._checkIfIgnoredScrollSource(targetActor) && !this._timeoutsHandler.getId(T6)) {
```
https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2216

This should fix https://github.com/home-sweet-gnome/dash-to-panel/pull/1939#issuecomment-1686780623

Reply:
Thanks, that works.